docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -922,14 +932,77 @@ These steps use `image-builder` to create a Ubuntu-based image for Nutanix AHV a
 
       * `--os`: `ubuntu`
       * `--hypervisor`: For Nutanix use `nutanix`
-      * `--release-channel`: Supported EKS Distro releases include 1-23, 1-24 1-25 1-26 and 1-27
+      * `--release-channel`: Supported EKS Distro releases include 1-23, 1-24, 1-25, 1-26 and 1-27.
       * `--nutanix-config`: Nutanix configuration file (`nutanix-connection.json` in this example)
 
       ```bash
       cd /home/$USER
       image-builder build --os ubuntu --hypervisor nutanix --release-channel 1-27 --nutanix-config nutanix-connection.json
       ```
 
+
+### Configuring OS version
+
+`image-builder` supports an `os-version` flag that allows you to configure which version of the OS you wish to build. If no OS version is supplied, it will build the default for that OS, according to the table below.
+
+<table style>
+    <thead align="center">
+        <tr>
+            <th><center>Operating system</th>
+            <th><center>Supported versions</th>
+            <th><center>Corresponding <code>os-version</code> value</th>
+            <th><center>Default <code>os-version</code> value</th>
+        </tr>
+    </thead>
+    <tbody align="center">
+        <tr >
+            <td rowspan=2><b>Ubuntu</b></td>
+            <td>20.04.1+<sup>*</sup></td>
```

Comment:
We should just say what this is right now, i like your note but we know what versions they are.

20.04.6?
22.04.1?
8.7?

Reply:
Done